extensions/ql-vscode/src/data-extensions-editor/data-extensions-editor-view.ts

```diff
@@ -97,6 +97,28 @@ export class DataExtensionsEditorView extends AbstractWebview<
     void extLogger.log(`Saved data extension YAML to ${modelFilename}`);
   }
 
+  protected async readExistingYaml(): Promise<void> {
```

Comment:
Is `readExistingModeledMethods()` a better name for this? We refer to YAML in function and message names, but that feels like an implementation detail. 

Also, usually with `readXXX` functions I expect something to be retuned. How about `loadXXX` instead?

Reply:
Sorry my comment was covering too many things - what do you think about updating messages as well to drop "yaml" and use "modeledMethods" e.g. `setExistingYamlData` -> `setExistingModeledMethods`?

Reply:
Good call! I've now moved all methods dealing with YAML from the webview to the extension. This also makes it easier to show an error message, so I've also added a simple error message when the data in the file is invalid.